src/bidiTab/bidiTab.ts

```diff
@@ -176,9 +181,9 @@ function createBidiServer(selfTargetId: string) {
 
     static #getErrorResponse(
       messageStr: string,
-      errorCode: Message.ErrorCode,
+      errorCode: ErrorCode,
       errorMessage: string
-    ): Message.OutgoingMessage {
+    ): ErrorResponse {
```

Comment:
This changed from `OutgoingMessage` to `ErrorResponse`. Is it intentional?

Reply:
Yes, because it's an error response that is returned.